.github/workflows/publish-docker.yaml

```diff
@@ -42,6 +42,19 @@ jobs:
           submodules: true
       - name: free disk space
         run: tools/github/free_disk_space.sh
+      - name: Free More Disk Space
+        run: |
+          echo "Disk space before cleanup:"
+          df -h
+          sudo rm -rf /usr/local/.ghcup
+          sudo rm -rf /opt/hostedtoolcache/CodeQL
+          sudo rm -rf /usr/local/lib/android/sdk/ndk
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf /usr/local/share/boost
+          sudo apt-get clean
+          echo "Disk space after cleanup:"
```

Comment:
How about move this shell script into https://github.com/apache/seatunnel/blob/353bbd21a1d242378a9dc7b420843e4745bdb5a7/tools/github/free_disk_space.sh#L23? So all workflow can benefit.